### ISSUE TITLE
Add TStream descendant classes

### ISSUE THREAD

**Issue by viniciusfbb** [Embarcadero/DelphiFMX4Python#35]
Add TStream descendant classes: TMemoryStream, TFileStream, TBytesStream...

I know the project's focus is only GUI and not portability of all Delphi classes, but some RTL classes are fundamental for some GUI features, such as classes that implement TStream, which can be used in a TBitmap.LoadFromStream, for example.

**Comment by Priyatham10** [comment]
Yes, our plan is to have the P4D core where we only wrap the RTL. Then WrapVCL and WrapFMX will branch out with P4D core as a dependency. What do you think of that plan @viniciusfbb ?
